features/FxA-80-eye-password-experiment/README.md

```diff
@@ -0,0 +1,147 @@
+* When the user clicks the control and the password is masked, the password becomes unmasked.
+* When the user clicks the control and the password is unmasked, the password becomes masked.
+
+## Press & Hold Down
+
+Possibly more familiar to Windows users, we can only show the password when the user presses and holds down on the icon (deprecated eye in animation for position only)
+
+<img alt="Press and Hold Down" src="eye-password-press.gif" width="525">
+
+### Acceptance Criteria
+
+* When the user hovers over the control and the password is masked, there should be a tooltip on the eye that says "Show password".
+* Windows-style: Only while the user is clicking the eye, does the password become unmasked.
+
+## Unmasked by Default
+
+A radical proposal that we should probably only test with the **Checkbox**  design.
+
+### Acceptance Criteria
+
+* The password fields are unmasked by default
+* When the user hovers over the control and the password is unmasked, there should be a tooltip that says "Hide password"
+* When the user hovers over the control and the password is unmasked, there should be a tooltip that says "Hide password"
+* When the user clicks the control and the password is masked, the password becomes unmasked.
+* When the user clicks the control and the password is unmasked, the password becomes masked.
+
+
+# Affordances
+### Checkbox
+
+<img alt="Checkbox Masked" src="eye-password-checkbox1.png" width="380">
+
+<img alt="Checkbox Unmasked" src="eye-password-checkbox2.png" width="380">
+
+### Eyeball
+
+<img alt="Eyeball Masked" src="eye-password-eyeball1.png" width="380">
+
+<img alt="Eyeball Unmasked" src="eye-password-eyeball2.png" width="380">
+
+The eye color matches our current smokey grey and blue colors.
+
+### Shades
+
+<img alt="Shades Masked" src="eye-password-shades1.png" width="380">
+
+<img alt="Shades Unmasked" src="eye-password-shades2.png" width="380">
+
+### Smiley
+
+<img alt="Smiley Masked" src="eye-password-smiley1.png" width="380">
+
+<img alt="Smiley Unmasked" src="eye-password-smiley2.png" width="380">
+
+
+### Monkey
+
+<img alt="Monkey Masked" src="eye-password-monkey1.png" width="380">
+
+<img alt="Monkey Unmasked" src="eye-password-monkey2.png" width="380">
+
+### Sunglasses
+
+<img alt="Sunglasses Masked" src="eye-password-sunglasses1.png" width="380">
+
+<img alt="Sunglasses Unmasked" src="eye-password-sunglasses2.png" width="380">
+
+
+# Success Criteria
+
+We will measure the events relative to the views of the Change Password section.
+```
+Settings › Change Password
+fxa.content.screen.settings.change_password
+```
+
+## Happiness
+Our goal is for users to not be weirded out by unmasking passwords. A signal for this is feedback on social media and user input.
+
+## Engagement
+Our goal is for users to unmask their passwords. A signal for this is that which design attracts the most usage.
```

Comment:
As I said in IRC, this seems like it could be an anti-goal. The necessity of unmasking a password points to a deeper problem, possibly even the user didn't enter their password correctly when signing up.